pkg/storage/unified/resource/keys.go

```diff
@@ -1,5 +1,18 @@
 package resource
 
+func verifyRequestKey(key *ResourceKey) *ErrorResult {
```

Comment:
not necessary for this PR.... but I spent 30mins banging my head where a nice error would have been better 🤦🏻